### DISCUSSION TITLE
Passing `-j` to GNU Make in a `custom_target`

### DISCUSSION THREAD

**Discussion by traxys** [mesonbuild/meson#13174] [Q&A] [answered]
I have a project where meson manages most of the targets, but this project contains a few Linux kernel modules, and as such they _must_ be compiled by calling `make` on the Linux kernel Makefile.
I'm pretty sure this means that they are built without any parallelism.

I tried to look into the jobserver support in ninja (or meson, but I think ninja is the right place for that) and did not find anything.
I'm pretty sure that running `make` with a jobserver is impossible in ninja, and even if it were I suppose it `meson` would require to output a rule that tells it to use the jobserver, is that right?

Or should this question be asked in `ninja`?

**Comment by eli-schwartz** [comment, marked as answer]
This would require https://github.com/ninja-build/ninja/pull/2260 to be able to do it. If implemented, we could generate a ninja file that runs make and has make ask ninja for job slots.

**Reply by eli-schwartz** [reply]
Really, it would just be thrilling if, among that set of PRs, ninja at least implemented client support so that people could build with ninja as part of a coordinated meta-build system such as buildroot or portage that compiles multiple ninja projects at the same time.

**Reply by traxys** [reply]
Ah I did not know that the terminology was `master`. I only found the client issue, but that did not seem applicable, but reading the comments I figured that there was no server either.
Thank you very much!
